### PR TITLE
Ont 435 for case 0.2.1

### PR DESCRIPTION
**Pull request by ajnelson-nist**
This is a re-do of [PR 54](https://github.com/casework/CASE/pull/54).

### CODE REVIEW

**Comment by ajnelson-nist** [comment]
@plbt5 , thanks for the review of PR 54.  This re-do PR (57) should only be merged after CP-38 is approved.

**Comment by ajnelson-nist** [comment]
@plbt5 , CP-38 has advanced far enough (it is just waiting on final change documentation to be posted) that you can review and merge this if you wish.

**Review by plbt5 — APPROVED** [review]
Checked and agreed

**Pull request merged**
